tests/gis_tests/gdal_tests/test_srs.py

```diff
@@ -386,3 +394,11 @@ def test_esri(self):
         self.assertIn('DATUM["D_North_American_1983"', srs.wkt)
         srs.from_esri()
         self.assertIn('DATUM["North_American_Datum_1983"', srs.wkt)
+
+    def test_srid(self):
+        """Testing if srid property returns top-level authority code"""
+        # Regression test for #34302
```

Comment:
This comment is unnecessary:
```suggestion
```

Reply:
I think the contributing docs must be updated in this regard (from [Patch Style](https://docs.djangoproject.com/en/4.1/internals/contributing/writing-code/submitting-patches/#patch-style)):

> Also, if some tickets are relevant to the code that you’ve written, mention the ticket numbers in some comments in the test so that one can easily trace back the relevant discussions after your patch gets committed, and the tickets get closed.

I pledge guilty for reading the docs. :innocent: